apps/components-e2e/src/components/quick-filter/quick-filter.e2e.ts

```diff
@@ -302,3 +303,49 @@ test('should check the show more with distinct values', async (testController: T
     .expect(getFilterfieldTags())
     .eql(['ValueValue 23']);
 });
+
+fixture('Quick Filter async')
+  .page('http://localhost:4200/quick-filter/async')
+  .beforeEach(async () => {
+    await resetWindowSizeToDefault();
+    await waitForAngular();
+  });
+
+test('should work with async data and handle distincts correctly', async (testController: TestController) => {
```

Comment:
I might be getting confused, but was the issue visible with this runthrough? I thought you had to select async three times and on the third time Linz was in the options list although it shouldn't have been? 

I know that the ids were already faulty, but I am not sure if the user already saw the issue with selecting a non async autocomplete in between. 
Sorry if i a can't remember right now.

Reply:
It was quite confusing, but yes the issue was visible with this runthrough. 
When selecting the first async filter completely, then choosing the same async filter again, we saw that the first filter was picked up correctly (ids were still fine). So after `Aut (async) Linz`, picking `Aut (async)` gave you the correct options of `Vienna` and `Graz`. 

When choosing a different filter after the first async, it showed the issue right away. So after `Aut (async) Linz`, picking `USA San Francisco` and then `Aut (async)` it gave the wrong options of `Linz`, `Vienna` and `Graz` whereas `Linz` should have been filtered out because it previously has been set.